Content.Client/Construction/UI/ConstructionMenuPresenter.cs

```diff
@@ -166,10 +166,23 @@ private void OnGridViewRecipeSelected(object? sender, ConstructionPrototype? rec
             if (_placementManager.IsActive && !_placementManager.Eraser) UpdateGhostPlacement();
             PopulateInfo(_selected);
         }
-
-        private void OnViewPopulateRecipes(object? sender, (string search, string catagory) args)
+		
+		private int checkfuzzysearch(string hostfield,string searchtext){// return an int for futureproofing, if you wanted to sort by likeness, or something. doesn't matter much now, ints are compatible with boolean logic, anyways.
+			int matchedtokens=0;
+			char[] str_seps={' ',':','.',',','/'}; //flatten punctuation.
+			string[] searchtokens = searchtext.Split(str_seps); //turn the search into tokens
+			
+			foreach (string stoken in searchtokens){
+				if(hostfield.Contains(stoken,StringComparison.OrdinalIgnoreCase)) matchedtokens++; //thanks chatGPT for helping me.
+			}
+			
+			return matchedtokens;
+		}
+		
+		
+        private void OnViewPopulateRecipes(object? sender, (string search, string catagory, bool fuzzysearch) args)
```

Comment:
fuzzysearch -> fuzzySearch, also might as well fix catagory while ur here

Reply:
that'd be unatomic and west would yell at me

Reply:
(yell at me to finish fission reactors, that is)

Reply:
fixed in commit